src/test/kotlin/uk/gov/dwp/dataworks/logging/LoggingUtilsTest.kt

```diff
@@ -0,0 +1,77 @@
+package uk.gov.dwp.dataworks.logging
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.classic.spi.IThrowableProxy
+import ch.qos.logback.classic.spi.ThrowableProxy
+import ch.qos.logback.classic.spi.ThrowableProxyUtil
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import org.apache.commons.text.StringEscapeUtils
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LoggingUtilsTest {
+    @Test
+    fun `semiFormattedTuples will format as partial JSON without matching key-value pairs`() {
+        assertThat("\"my-message\"").isEqualToIgnoringWhitespace(semiFormattedTuples("my-message"))
+    }
```

Comment:
why ignoring whitespace? how it concatenates is important?

Reply:
Well noted. Will update